test/runtests.jl

```diff
@@ -455,6 +455,22 @@ end
     end
 end
 
+@testset "Auditor - absolute paths" begin
+    prefix = Prefix(tempname())
+    try
+        sharedir = joinpath(prefix.path, "share")
+        mkpath(sharedir)
+        open(joinpath(sharedir, "foo.conf"), "w") do f
+            write(f, "share_dir = \"$sharedir\"")
+        end
+
+        # Test that `audit()` warns about an absolute path within the prefix
+        Compat.@info("Expecting a warning about share/foo.conf:")
+        BinaryBuilder.audit(prefix)
+    finally
```

Comment:
Didn't you mean `rescue`? :trolleybus:

Reply:
we're not rich enough to afford that kind of syntax

Reply:
maybe if we sent more of our coders into the mines, they might come back out with some more gems like those